airflow/ui/src/queries/useCreateBackfill.ts

```diff
@@ -0,0 +1,86 @@
+/*!
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import { useState } from "react";
+
+import { useBackfillServiceCreateBackfill } from "openapi/queries";
+import type { CreateBackfillData } from "openapi/requests/types.gen";
+import { toaster } from "src/components/ui";
+
+export const useCreateBackfill = ({ onSuccessConfirm }: { onSuccessConfirm: () => void }) => {
+  const [dateValidationError, setDateValidationError] = useState<unknown>(undefined);
+  const [error, setError] = useState<unknown>(undefined);
+
+  const onSuccess = () => {
+    toaster.create({
+      description: "Backfill jobs have been successfully triggered.",
+      title: "Backfill generated",
+      type: "success",
+    });
+    onSuccessConfirm();
+  };
+
+  const onError = (_error: unknown) => {
+    setError(_error);
+  };
+
+  const { isPending, mutate } = useBackfillServiceCreateBackfill({ onError, onSuccess });
+
+  const createBackfill = (data: CreateBackfillData) => {
+    const DataIntervalStart = new Date(data.requestBody.from_date);
+    const DataIntervalEnd = new Date(data.requestBody.to_date);
```

Comment:
```suggestion
    const dataIntervalStart = new Date(data.requestBody.from_date);
    const dataIntervalEnd = new Date(data.requestBody.to_date);
```

Normal variables start with a lowercase letter

Reply:
Ack